templates/observability/managed.html

```diff
@@ -0,0 +1,577 @@
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">24/7 monitoring</h3>
+            <div class="p-matrix__desc">
+              <p>Active management and break/fix response by Canonical engineers.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">High availability</h3>
+            <div class="p-matrix__desc">
+              <p>Fault-tolerant and ready for real-world challenges. Your monitoring tools will be there for you when you need them.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Security first</h3>
+            <div class="p-matrix__desc">
+              <p>Industry-wide security best practices and compliance certifications to keep your cloud secure at all times.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Flexible pricing</h3>
+            <div class="p-matrix__desc">
+              <p>Transparent and predictable pricing to suit your needs and help you grow your cloud with predictable costs.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">No lock-in</h3>
+            <div class="p-matrix__desc">
+              <p>Build a flexible and agnostic cloud with no lock-in and leverage a simplified open-source approach. We give you the keys when you are ready to drive.</p>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<div class="u-fixed-width">
+  <hr>
+</div>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h3>Reliable monitoring tools you can count on</h3>
+      <p>All Canonical managed services come with Extended Security Maintenance: future-proofed open-source observability run at scale, reliably.</p>
+      <p>
+        <a href="/observability/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <ul class="p-matrix">
+      <li class="p-matrix__item u-align--center">
+        {{
+          image(
+          url="https://assets.ubuntu.com/v1/d12bc087-apps_promethus.png",
+          alt="promethus",
+          width="144",
+          height="90",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+        <div class="p-matrix__content u-align--center">
+          <div class="p-matrix__desc">
+            <p class="p-muted-heading">HA-READY PROMETHEUS AND ALERTMANAGER INSTANCES</p>
```

Comment:
In the past we have left these lower-case as the style will make it uppercase. But I see on the doc it is in uppercase too, so I guess it is up to you.